cmake/ccache.cmake

```diff
@@ -3,7 +3,7 @@
 
 if(USE_CCACHE STREQUAL "0")
 else()
-  find_program(CCACHE_FOUND ccache)
+  find_program(CCACHE_FOUNDX ccache)
```

Comment:
what does this change do?

Reply:
=> @nashif

Reply:
yep, leftover from debugging., will remove